core/headerchain.go

```diff
@@ -1269,15 +1269,20 @@ func (hc *HeaderChain) CalcMaxBaseFee(block *types.WorkObject) (*big.Int, error)
 		return nil, errors.New("parent cannot be found in the CalcMaxBaseFee")
 	}
 	parentBaseFee := parent.BaseFee()
+
+	minBaseFee := hc.CalcMinBaseFee(block)
+	blockBaseFee := block.BaseFee()
+	if blockBaseFee.Cmp(common.Big0) == 0 {
```

Comment:
Should this be blockBaseFee.Cmp(minBaseFee) < 0 ?

Reply:
done